doc/classes/Path2D.xml

```diff
@@ -9,9 +9,55 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="is_debug_global_enabled" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if the global path debug is enabled.
+			</description>
+		</method>
+		<method name="set_debug_global_enabled">
+			<return type="void" />
+			<param index="0" name="enabled" type="bool" />
+			<description>
+				If [code]true[/code], enables the global path debug.
```

Comment:
Heavily needs a proper explanation, over "enables the global path debug". Also a link to its associated setter. Something along the lines of:
```suggestion
				If [code]true[/code], enables debugging for all [Path2D] in the scene tree. See also [method set_debug_global_enabled].
```
I cannot say for certain what this means, as of writing this.